tests/cluster/tests/30-reliable-meet.tcl

```diff
@@ -0,0 +1,54 @@
+set do_not_join_cluster_nodes 1
```

Comment:
Can you move this test to `test/unit/cluster` instead? It's filled with tests that actually run as part of the CI and use less custom functionality. It also will allow you to more easily start a net node and have it join without having to rely on this weird global.